services/gateway/apis/http-version3/swagger/definitions/networkStatus.json

```diff
@@ -92,41 +101,49 @@
                                 "dpos",
                                 "fee",
                                 "random",
-                                "legacy"
+                                "legacy",
+                                "interoperability"
                             ]
                         },
                         "description": "Array of all the current registered modules"
                     },
                     "moduleCommands": {
```

Comment:
The example currently returns `moduleCommands` as a nested array. Please fix it.

Reply:
applied.